src/components/test-slot/test-outcome/test-outcome.ts

```diff
@@ -128,7 +130,11 @@ export class TestOutcomeComponent implements OnInit {
 
   debriefTest() {
     this.store$.dispatch(new ActivateTest(this.slotDetail.slotId));
-    this.navController.push(PASS_FINALISATION_PAGE);
+    if (this.activityCode === ActivityCodes.PASS) {
```

Comment:
Like how explicit you are over here. It just made me thinking how could this be done differently.

`const pageName = this.activityCode === ActivityCodes.PASS ? PASS_FINALISATION_PAGE : NON_PASS_FINALISATION_PAGE;`
`this.navController.push(pageName);`

You decide if you want to change it, outcome is the same, just different style.